cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/UserManagementTest.scala

```diff
@@ -54,6 +54,14 @@ Set a new password for a user. This user will not be required to change this pas
 ###assertion=update-one parameters=update
 //
 
+ALTER USER alice IF EXISTS SET PASSWORD CHANGE REQUIRED
+###
+
+Required to change the password on the next login for a user, if the user exists.
```

Comment:
This sentence feels a bit weird... 🤔

Reply:
I know, I did not come up with anything that sounded good, if you have any idea, please help :P

Reply:
```suggestion
If the specified user exists, force this user to change their password on the next login.
```